src/chef-server-ctl/chef-server-ctl.gemspec

```diff
@@ -52,7 +51,7 @@ Gem::Specification.new do |spec|
   # tools we bundle in the chef-server install and include here so we can have a single Gemfile.lock
   # for the overall chef-server "app"
   spec.add_runtime_dependency "knife-ec-backup"
-  spec.add_runtime_dependency "chef_fixie"
+  spec.add_runtime_dependency "chef_fixie_shahid" #Note: replace with chef_fixie after fixing the chef_fixie pipelines
```

Comment:
1.0 is out so you can make this a dep on >= 1.0 and we should be good

Reply:
updated